src/callinput.c

```diff
@@ -844,9 +844,11 @@ int callinput(void) {
 
 	    // Ctrl-B (^B), send spot to DX cluster.
 	    case CTRL_B: {
-		announcefilter = 0;
-		cluster = CLUSTER;
-		add_cluster_spot();
+		if (!nopacket && packetinterface > 0) {
+		    announcefilter = 0;
+		    cluster = CLUSTER;
```

Comment:
I just kept the old behavior. I assume the idea was to enable cluster display so you can see if your announcement was going through.

Line 848 is configuring the level of details which is shown from the cluster connection. It is set to display all information.

But it is the same here for me: I use mostly the :MAP mode also and the switch to :CLUster mode is irritating.

Maybe it would be better to respect the decision of the operator what to display (:MAP/:CLUster) and drop line 848 and 849 completely.

Reply:
Ok, I have the same feeling. Dropped the switch in the last commit.

I am not sure if `announcefilter` is used anyway. So that needs to be revisited anyway later.